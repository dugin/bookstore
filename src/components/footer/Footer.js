import React from 'react';


const Footer = () => {

    const styles = {
        textAlign: 'center',
        marginTop: '20px'
    };
    return (
        <div style={styles}>
            <p>Developed by Rodrigo Dugin</p>
        </div>
    );
}

export default Footer;
