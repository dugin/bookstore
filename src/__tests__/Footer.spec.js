import React from 'react';
import {shallow} from 'enzyme';
import Footer from '../components/footer/Footer';
import toJson from 'enzyme-to-json';

describe('Footer ', ()=>{

    const footer = shallow(<Footer/>);

    it('renders properly', ()=>{

        expect(toJson(footer)).toMatchSnapshot();
    });

    it('should contain a paragraph with the text `Developed by Rodrigo Dugin`', ()=>{

        expect((footer).find('p').contains('Developed by Rodrigo Dugin')).toBe(true);
    })
})