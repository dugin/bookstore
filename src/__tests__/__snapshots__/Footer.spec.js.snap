// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer  renders properly 1`] = `
<div
  style={
    Object {
      "marginTop": "20px",
      "textAlign": "center",
    }
  }
>
  <p>
    Developed by Rodrigo Dugin
  </p>
</div>
`;
